shell.nix

```diff
@@ -2,10 +2,15 @@ let
   sources = import ./nix/sources.nix;
   pkgs = import sources.nixpkgs {};
   unstable-pkgs = import sources.nixpkgs-unstable {};
+  inherit (pkgs) lib stdenv;
 in
 pkgs.mkShell {
   name = "rawfile-shell";
 
+  NIX_LD_LIBRARY_PATH = lib.makeLibraryPath [
```

Comment:
Is this a special env name for nix?

Reply:
Yes, Checkout https://github.com/nix-community/nix-ld/blob/main/examples/masterpdfeditor.nix as a full example